From b979ad6522e3c47c7e16d1c028530cf35f0e0756 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Wed, 28 Jun 2023 12:34:35 +0300
Subject: [PATCH] Revert "[SW-149942] efuse: add init as part of general flow"

preboot is used in preboot-rtos project, so this
patch is not relevant as it is a dead code.
This reverts commit da0f484970e66f0cbbc9fec873318662844e25cd.

Change-Id: I04c230cb1a5d7b7c0a6ee4eb80d0a5384738f7cf
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index 030d8ab..9d6504d 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -19,7 +19,6 @@
 #include "pll.h"
 #include <boot_if.h>
 #include <pci.h>
-#include <efuse.h>
 #include <hl.h>
 
 struct device_ca53_cfg ca53_cfg;
@@ -341,7 +340,6 @@
 
 	preboot_sync_btl();
 	hl_uart_init();
-	preboot_fuse_init();
 	spi_flash_prot_early();
 	ppboot_security_init();
 	hl_pcie_init();
